java/src/Ice/src/main/java/com/zeroc/IceInternal/ProxyOutgoingAsyncBaseI.java

```diff
@@ -373,8 +522,7 @@ protected void prepare(java.util.Map<String, String> ctx) {
 
   protected final com.zeroc.Ice._ObjectPrxI _proxy;
   protected RequestHandler _handler;
-  protected com.zeroc.Ice.OperationMode _mode;
-  protected int _proxyMode;
```

Comment:
I also removed our caching of `proxyMode` here... We were only using it in 2 spots, to check if something was batch or not. Now that we have a helpful `isBatch` function, no point storing the int and re-writing that logic.